radio/src/gui/colorlcd/model_select.cpp

```diff
@@ -208,22 +208,25 @@ class ModelCategoryPageBody : public FormWindow
           Menu *menu = new Menu(parent);
           if (model != modelslist.getCurrentModel()) {
             menu->addLine(STR_SELECT_MODEL, [=]() {
-              // we store the latest changes if any
+              bool modelConnected = TELEMETRY_STREAMING() &&
```

Comment:
I think "OR" could be valid because non telemetry receivers probably are not streaming telemetry.

Reply:
I don't think so in this instance, as OR would also trigger when there is no telemetry and setting is disabled, because it is an inverted test. 

Please check to see if I missed something (note: I'm taking into account real end state of test ... having `!` and not having `!`. However, the more practical test is on hardware, and it _seems_ to be working as intended... with telemetry active, confirm warning shows, with RSSI warning disabled and telemetry active, no confirm warning, no telemetry, no confirm warning. 
```
TELEMETRY_STREAMING() && !disableRssiPoweroffAlarm  | 
1                     &&  1 (not disabled)          | T
1                     &&  0 (disabled)              | F
0                     &&  1 (not disabled)          | F
0                     &&  0 (disabled)              | F


TELEMETRY_STREAMING() || !disableRssiPoweroffAlarm  | 
1                     ||  1 (not disabled)          | T
1                     ||  0 (disabled)              | T
0                     ||  1 (not disabled)          | T
0                     ||  0 (disabled)              | F


TELEMETRY_STREAMING() ||  disableRssiPoweroffAlarm  | 
1                     ||  1 (disabled)              | T
1                     ||  0 (not disabled)          | T
0                     ||  1 (disabled)              | T
0                     ||  0 (not disabled)          | F
```

Reply:
I think your code is correct.

Reply:
Phew... I have always hated truth tables when combined with inversions... so always a chance stuffed it up ;) Thanks! :)